### DISCUSSION TITLE
copilot primer workshop removed?

### DISCUSSION THREAD

**Discussion by tyoc213** [aws/copilot-cli#5597] [Q&A] [answered]
I think I learned to use with this which now is 404 (but still kind of listed) https://catalog.us-east-1.prod.workshops.aws/workshops/bbe33a01-4c98-4cfc-b859-d55d699419b8/en-US but it is no longer available, I was gonna use it as base to introduce some friends to the IaC

![image](https://github.com/aws/copilot-cli/assets/506234/b0f76b50-21a1-4b49-9f40-f45c4ec5d802)

from https://workshops.aws/

**Comment by iamhopaul123** [comment]
It seems like the primer one is still there but the other one is missing right now. We actually have no control over these workshop sessions. I'm trying  to reach out through [Contact/feedback](mailto:aws-workshops@amazon.com) at the bottom of the page.

**Reply by iamhopaul123** [reply, marked as answer]
Update: the workshop team told me they are currently actively working on all the workshops that were accidentally removed. The ETA right now would be March. Sorry for the inconvenience.

**Reply by tyoc213** [reply]
haha, not fun, wanted to introduce them in January, not in march xD ... @iamhopaul123 any other nice resource for introducing some projects to some random guys out there that you can recommend?

**Reply by iamhopaul123** [reply]
Sure! We have some more workshops [here](https://aws.github.io/copilot-cli/community/guides/#workshops) and this [ecs workshop](https://ecsworkshop.com/microservices/) is good for new users.
